test/sprinkler_web/mqtt/handler_test.exs

```diff
@@ -24,5 +26,11 @@ defmodule SprinklerWeb.Mqtt.HandlerTest do
     test "terminate/2 returns ok" do
       assert Handler.terminate("termination is imminent", []) == :ok
     end
+
+    test "handle_message/3 when telemetry is received" do
+      Phoenix.PubSub.subscribe(Sprinkler.PubSub, @telemetry)
+      Handler.handle_message(["rs", "Arduino", "telemetry"], "25", [])
```

Comment:
I think in reality this will be a JSON, we can change it once we define how we communicate with the devices.
```suggestion
      Handler.handle_message(["rs", "Arduino", "telemetry"], "{\"temp\":24}", [])
```

Reply:
Great, so i'm not committing this just now.

Reply:
Don't we have a way of parsing that json and using that instead of the raw string?